main/main.cpp

```diff
@@ -3191,6 +3191,10 @@ Error Main::setup2(bool p_show_boot_logo) {
 			}
 
 			id->set_emulate_mouse_from_touch(bool(GLOBAL_DEF_BASIC("input_devices/pointing/emulate_mouse_from_touch", true)));
+
+			if (editor) {
```

Comment:
Should this also include the `project_manager`?

Reply:
Project manager seems to be unaffected by the setting but i can include it if necessary